module-1/src/Components/MovieForm/MovieForm.tsx

```diff
@@ -30,71 +130,115 @@ export const MovieForm = ({ title, movie, closeCallback }: MovieFormProps) => {
           />
         </svg>
       </div>
+
       <h1 className="ml-5 mb-2 text-white text-2xl font-Montserrat">{title}</h1>
-      <form onSubmit={(e) => handleSubmit(e)}>
+      <form onSubmit={handleSubmit(onSubmit)}>
         <div className="grid grid-cols-2 gap-3 mx-5 pb-3">
           <div className=" grid col-auto">
             <label
               className="text-custom_pink font-Montserrat "
-              htmlFor="Title"
+              htmlFor="title"
             >
               Title
             </label>
             <input
+              {...register("title", { required: true })}
               className="bg-custom_light_gray text-white p-1"
               type="text"
-              name="Title"
-              id="Title"
-              value={movie?.movieName}
+              name="title"
+              id="title"
+              value={titleString}
+              onChange={(e) => setTitleString(e.target.value)}
             />
+            {errors.title?.type === "required" && (
```

Comment:
You have a bunch of input+error state groups. You can extract them into a component not to repeat yourself every time you need to add an input.